sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -110,6 +110,47 @@ void peekMessage() {
             .verifyComplete();
     }
 
+    /**
+     * Verifies that we can schedule and peek a message.
+     */
+    @Test
+    void scheduleMessage() {
+        // Arrange
+        final String messageId = UUID.randomUUID().toString();
+        final String contents = "Some-contents";
+        final ServiceBusMessage message = TestUtils.getServiceBusMessage(contents, messageId, 0);
+        int enqueDelayInSec = 2;
+        // Assert & Act
+        StepVerifier.create(sender.scheduleMessage(message, Instant.now().plusSeconds(enqueDelayInSec))
```

Comment:
The variable creations like Instant.now.plus.. and Duration.ofSeconds(enqeueDe...) are a part of your "Arrange" section. The Act part is usually, "I set up all my things, now let me invoke it."